Aptoide/aptoide-client-v8
# [ASV-2156] AB Testing similar apps experiment fix

**What does this PR do?**

  Attempt to solve the issue regarding Rakam uneven groups (control group has much more hits then appc_bundle), which I cannot reproduce but the analytics says there is a problem.
  Also turns on this experiment in Prod so we can get numbers from production versions.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] SimilarAppsExperiment.kt (dev/prod/etc)

**How should this be manually tested?**

  Go to an app view of an app with a Similar apps bundle (check group with - /api/v1/assignments/applications/Android/experiments/ASV-2053-SimilarApps/users . Click Install > See Impression/conversion to Rakam).

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-2156](https://aptoide.atlassian.net/browse/ASV-2156)


**Code Review Checklist**
